### ISSUE TITLE
Add an option to display title either logo or logoDark are configured

### ISSUE THREAD

**Issue by BattlefieldDuck** [retypeapp/retype#26]
Thank you for this beautiful docs project, I really love it😍.

My current issue is I want to display both title and logo together on the docs, but it is not displayed when logo or logoDark is configured according to (https://retype.com/configuration/project/#title).

My proposed solution
```
{
  "branding": {
    "title": "Example.com",
    "logo": "static/logo.png",
    "hideTitle": false
  }
}
```

Thank you.

**Comment by geoffreymcgill** [comment]
Thanks for the feedback and suggestion. I completely agree that rendering both the `title` and `logo|logoDark` together should be an options. I have wanted us to add this functionality for a while but we've just been super focused on some other tasks. 

I will make this a priority. I'm not sure it can be done in time for the upcoming `v1.6.0` release, but I'll try and get it in there. 

Thanks again for the feature request.

**Comment by geoffreymcgill** [comment]
The functionality to configure both a `logo` and a `title` has been implemented in Retype `v1.6.0` and is now publicly available.

We also added a new `logoAlign` config with possible options being `"left"` or `"right"`. Default is `left`. If set to `right`, the logo will be aligned to the right of the `title`. 

More documentation is being added, although the functionality is straightforward to understand and should _just work_ as you expect. If it doesn't, please let us know. 

I will add a link to the new documentation once it's published to https://retype.com/configuration/project/#branding.

@BattlefieldDuck Thanks again for the feature request. Hope this helps.
